internal/resource/resource.go

```diff
@@ -336,7 +336,7 @@ func NewResource(ctx context.Context, renv *readiness.Env, slice *apiv1.Resource
 			name = "default"
 		}
 
-		check, err := readiness.ParseCheck(renv, value)
+		check, err := readiness.ParseCheck(readiness.DefaultEnv, value)
```

Comment:
Not worth the injection?

Reply:
hmm maybe we should just use the default env from ParseCheck and make it private. Will investigate.

Reply:
Refactored some things to encap the cel env entirely within the readiness package